package.json

```diff
@@ -29,5 +29,8 @@
     "lint-staged": "^10.2.2",
     "prettier": "^2.0.5",
     "typescript": "^3.8.3"
+  },
+  "dependencies": {
+    "@vtex/danger": "^0.2.7"
```

Comment:
I have a question: Shouldn't this go under `devDependencies` ?

Reply:
It's a good question. Let me check if your suggestion works. I don't added on `devDependences` because it install when action run. But I can try 😄

Reply:
Your suggestion really works 🏆